### ISSUE TITLE
Sound problem PSP

### ISSUE THREAD

**Issue by Kystan** [technicaljicama/godot-psp#14]
PSP 3000

The problem with the sound is its crackling. Maybe I should somehow adjust the imported sound, if so then let me know! I import by transferring the file with the mouse from the desktop directly to the editor (I use StreamPlayer)
[i-still-dream-about-her.zip](https://github.com/PSVita-Dev/godot-psp/files/12809214/i-still-dream-about-her.zip)

there were no problems with sound on the emulator

**Comment by technicaljicama** [comment]
I found out that the sound is only an issue on a real PSP and not even on adrenaline. I don't know why, maybe the PSP CPU is doing too much, I might do a few adjustments in the future

**Comment by Kystan** [comment]
thank you very much, this is very important for my development! Thank you for what you are doing :3

**Comment by technicaljicama** [comment]
Yeah.. I don't see why this happens... Maybe someone else may know it, but for now I'll guess that the cpu is doing too much already. (You maybe know it from some bad emulators that the sound is choppy)

**Comment by Kystan** [comment]
I think it's because of the bass, I removed them with https://vocalremover.org/ru/splitter-ai and the crackling has become much less, but it is still there

**Comment by technicaljicama** [comment]
Maybe a real PSP handles volume ranges a bit differently for some reason.. Interesting find!

**Comment by Kystan** [comment]
what is interesting, but also obvious, is that the sound in the headphones is with the same crackle

**Comment by technicaljicama** [comment]
I might increase the volume db times 3 sometime

**Comment by antim0118** [comment]
> The problem with the sound is its crackling. 

Your sound has cracking. I guess, that's the reason of the problem.

![image](https://github.com/technicaljicama/godot-psp/assets/22983870/f1763bd5-6e31-49b6-93a3-5929327b8dd5)

**Comment by hyp3rb0yy** [comment]
I have the same issue. It appears to happen when audio isn't played through the psp's media engine. I figured it out while testing development on Lua Player Euphoria. There, all files except .wav playback using the PSP Media Engine and only .wav fiels sound crunchy and "crackling", even on ppsspp. But with godot, this issue seems to not appear on the emulator, which is strange.

**Comment by antim0118** [comment]
> I have the same issue. It appears to happen when audio isn't played through the psp's media engine. I figured it out while testing development on Lua Player Euphoria. There, all files except .wav playback using the PSP Media Engine and only .wav fiels sound crunchy and "crackling", even on ppsspp. But with godot, this issue seems to not appear on the emulator, which is strange.

Nah, that's not the reason. The reason of cracking sound is threading issue. This problem not appears on ppsspp because emulator is a way faster than console.
